packages/react/src/time-input/TimeInput.stories.tsx

```diff
@@ -41,7 +41,7 @@ function Render(props: TimeInputProps) {
 		setValue(timeValue);
 	};
 
-	const invalid = value && !isValidTime(value.raw);
+	const invalid = value && !isValidTime(value.value);
```

Comment:
`value.value` 😅 

I'm sure it's fine. I don't have an alternative anyway.

Reply:
Same as what Combobox does effectively